bolt-cli/src/commands/operators.rs

```diff
@@ -254,6 +254,34 @@ impl OperatorsCommand {
                         warn!(?address, "Operator not registered");
                     }
 
+                    // Check if operator has collateral
+                    let mut total_collateral = Uint::from(0);
+                    for (name, address) in deployments.collateral {
+                        let stake =
+                            match bolt_manager.getOperatorStake(address, address).call().await {
+                                Ok(stake) => stake._0,
+                                Err(e) => match try_parse_contract_error::<
+                                    BoltEigenLayerMiddlewareErrors,
+                                >(e)?
+                                {
+                                    BoltEigenLayerMiddlewareErrors::KeyNotFound(_) => Uint::from(0),
+                                    other => unreachable!(
+                                        "Unexpected error with selector {:?}",
+                                        other.selector()
+                                    ),
+                                },
+                            };
+                        if stake > Uint::from(0) {
+                            total_collateral += stake;
+                            info!(?address, token = %name, amount = ?stake, "Operator has collateral");
+                        }
+                    }
+                    if total_collateral > Uint::from(0) {
+                        info!(?address, total_collateral=?total_collateral, "Total operator collateral");
+                    } else {
+                        warn!(?address, "Operator has no collateral");
+                    }
+
```

Comment:
This is good, I'd also write to stdout whether the operator is "active" i.e. its total collateral is greater than 1 ether.

Reply:
ok to hardcode this or should >1ETH be retrieved from somewhere? Is it solidified?

Reply:
for now ok to hardcode it imo

Reply:
You can either read it from here https://github.com/chainbound/bolt/blob/16152bde6129e91ab01a991bd87651cd37c2c2a5/bolt-contracts/src/contracts/BoltParametersV1.sol#L56 or hardcode it. As you prefer.